{{cookiecutter.project_shortname}}/run-tests.sh

```diff
@@ -3,7 +3,7 @@
 
 pipenv check && \
 pipenv run pydocstyle {{ cookiecutter.package_name }} tests docs && \
-pipenv run isort -rc -c -df && \
+pipenv run isort -c --df && \
```

Comment:
Normal: This time was just a `-` what broke our tests. We could have more resilience if we expand the commands. Also it would help new developers. Instead of going to the terminal and running `isort --help` they would have a clearer idea that `-c` meand "only check the files". Something like:

```suggestion
pipenv run isort --check-only --diff && \
```

But okay `-rc` got removed anyway. So expanding is not 100% resilient since still has the removal problem.

WDYT @ntarocco @lnielsen @slint @jrcastro2

Reply:
I'm also generally in favor of using the expanded/full name of options in scripts, since it's a form of documentation compared to the "cryptic" short versions which are nice for being quick when typing on an everyday basis.